apps/cowswap-frontend/src/modules/tokensList/pure/TokenTags/index.tsx

```diff
@@ -12,51 +13,58 @@ interface TagInfo {
   name: string
   description: string
   icon?: string
+  color?: StatusColorVariant
 }
 
-enum Tags {
-  UNSUPPORTED = '0',
-  GAS_FREE = '1',
-}
+export type TokenTagType = keyof typeof TOKEN_TAGS
 
-const TOKEN_TAGS: Record<Tags, TagInfo> = {
-  [Tags.UNSUPPORTED]: {
+const TOKEN_TAGS: Record<string, TagInfo> = {
+  unsupported: {
     name: 'Unsupported',
     description:
       'This token is unsupported as it does not operate optimally with CoW Protocol. Please refer to the FAQ for more information.',
     id: '0',
+    color: StatusColorVariant.Warning,
   },
-  [Tags.GAS_FREE]: {
-    name: 'Gas-free approval',
+  'gas-free': {
+    name: 'Gas-free',
     icon: ICON_GAS_FREE,
     description: 'This token supports gas-free approvals. Enjoy! 🐮',
     id: '1',
+    color: StatusColorVariant.Success,
+  },
+  circle: {
+    name: 'Circle Native',
+    description: 'Token officially issued by Circle',
```

Comment:
I thought this data would come from the token list tag definition?

Reply:
Yeah actually good call. Somehow I decide to rename it, but obviously we should do it in the token list. Will revert to use directly from the token list instead.